### PR TITLE
ensure qos is int

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: lib/configService.js [hunk 1]
```diff
@@ -108,7 +108,7 @@ function processEnvironmentVariables() {
     }
 
     if (process.env.IOTA_MQTT_QOS) {
-        config.mqtt.qos = process.env.IOTA_MQTT_QOS;
+        config.mqtt.qos = parseInt(process.env.IOTA_MQTT_QOS);
```
Comment (fgalan) [inline lib/configService.js]: What would happen if process.env.IOTA_MQTT_QOS is not an integer? Would parseInt() raises an exeception and the IoTAgent exits in a proper manner?

Comment (AlvaroVega) [reply lib/configService.js]: String '1' is translated to integer=1

If IOTA_MQTT_QOS is not and integer then parseInt do not raises an exception, just return NaN

Comment (AlvaroVega) [reply lib/configService.js]: In that case NaN in QoS is interpreted as QoS 0

Comment (fgalan) [reply lib/configService.js]: What about if the configuration comes from config.js file? It seems this parseInt() works only when the configuraiton is provided by env var.

Maybe it is better to do the check after env var has been processed, in the resulting config.mqtt.qos variable.

Comment (AlvaroVega) [reply lib/configService.js]: Where can we check that? If I'm not wrong other config values are not cheched in that sense.

Comment (AlvaroVega) [reply lib/configService.js]: Here https://github.com/telefonicaid/iotagent-json/blob/master/lib/bindings/MQTTBinding.js#L93 should be tested

Comment (fgalan) [reply lib/configService.js]: Yes... probably that point would ensure that QoS is correctly processed, not matter if the configuration was provided by config.js or by env vars. I'd suggest to move the check there.

**Pull request merged**